sections/4-development.tex

```diff
@@ -12,13 +12,9 @@ \section{Development Model}
     \item All new features must be accompanied with documentation.
     This includes code comments, formal documentation, and gallery examples.
     \item Test code must be provided with coverage reports.
-    \item Finally, all code must be reviewed and accepted by at least two members of the developer community before it is integrated into the codebase.
+    \item Finally, all code must be within scope and be reviewed and accepted by at least two members of the developer community before it is integrated into the codebase.
 \end{enumerate}
-These requirements are imposed on every single contribution.
-
-The \sunpyproj adheres to a schedule of two releases of \sunpypkg per year -- a Long Term Support (LTS) release and a short-support or non-LTS release. 
-LTS releases will be supported for 12 months or until the next LTS release; non-LTS releases will be supported for 6 months or until the next release. Support periods can be extended beyond the requirements here if needed.
```

Comment:
We don't want this somewhere else?

Reply:
There is a whole release section. This text is repeated there.